bionic/flow.py

```diff
@@ -99,6 +105,13 @@ def define_entity(self, entity_def):
         if entity_def.name in self.entity_defs_by_name:
             raise AlreadyDefinedEntityError(entity_def.name)
 
+        if entity_is_internal(entity_def.name) and entity_def.can_persist:
+            message = f"""
+            Attempted to set @persist to True for Bionic internal
```

Comment:
Can we provide more clarity on what the user needs to do to fix this, similar to the warning we have below?